src/components/parameter/ParameterEdit.vue

```diff
@@ -405,23 +591,49 @@ export default Vue.extend({
         state.options = props.param['options'];
         state.param = props.param as ModelParameter | Parameter;
         state.expertMode = !state.param.isConstant();
-        state.items[1].visible = 'arrayInput' === state.options.input;
-        state.items[2].visible = ['valueSlider', 'arrayInput'].includes(
-          state.options.input
-        );
       } else {
         state.options = props.options;
       }
+      state.options.errorMessages = [];
+      showMenuItems();
     };
 
     /**
-     * (Re-)Checks if the warning about the simulation resolution should be
-     * shown.
+     * Close message text.
      */
-    function updateResolutionWarningState() {
-      state.showResolutionWarning =
-        state.options.label === 'simulation resolution' && state.value < 1;
-    }
+    const closeMessage = () => {
+      if (state.timeoutId) {
+        clearTimeout(state.timeoutId);
+      }
+      state.message = '';
+    };
+
+    /**
+     * Rules for value validation.
+     */
+    const rules = [
+      (value: number) => {
+        if (state.timeoutId) {
+          clearTimeout(state.timeoutId);
+        }
+        let messageType: string = '';
+        if (state.options.unit && state.options.unit === 'ms' && value < 0) {
+          messageType = 'error';
+          state.message = `The ${state.options.label} cannot be negative.`;
+        } else if (state.options.rules !== undefined) {
+          state.options.rules.forEach((rule: string[]) => {
+            state.message = eval(rule[0]) ? rule[1] : '';
+            messageType = eval(rule[0]) ? rule[2] : '';
+          });
+        }
+        if (state.message.length > 0) {
+          state.timeoutId = setTimeout(() => {
+            state.message = '';
+          }, 5000);
```

Comment:
I would suggest to add a slightly greater timespan, since the main user group of NEST Desktop consists of persons having not much IT experience.
```suggestion
          }, 7500);
```